pkg/asset/installconfig/pullsecret.go

```diff
@@ -25,7 +25,7 @@ func (a *pullSecret) Generate(asset.Parents) error {
 		&survey.Question{
 			Prompt: &survey.Input{
 				Message: "Pull Secret",
-				Help:    "The container registry pull secret for this cluster, as a single line of JSON (e.g. {\"auths\": {...}}).",
+				Help:    "The container registry pull secret for this cluster, as a single line of JSON (e.g. {\"auths\": {...}}).\n\nYou can get this secret from https://try.openshift.com",
```

Comment:
users were complaining https://github.com/openshift/installer/pull/663/files#diff-e88e63c9df89ea6d2969694596325266L44 was insufficient. now this is more obscure. We should add/point to some docs on getting the pull secret process.

Reply:
try.openshift.com is fairly tightly scoped to this.  Not sure about the old account.coreos.com.  I'll go through the process again and see if there's anything that seems non-obvious.

Reply:
So here's what the try.openshift.com flow looks like:

![login](https://user-images.githubusercontent.com/209920/48645679-40226a00-e99b-11e8-8be8-42b67e9d2a4e.png)

![secret](https://user-images.githubusercontent.com/209920/48645787-85469c00-e99b-11e8-93a9-2a0d9c6ea67b.png)

I don't know what text I could add to make that process easier, except that it would be nice if the secret was on a single line (I think @crawford has already asked for that) or if the web page suggested downloading the secret instead of copy/pasting it.  Thoughts?

Reply:
Even the old process was 2 steps:
![screenshot from 2018-11-16 12-53-45](https://user-images.githubusercontent.com/3481311/48646958-49add100-e99f-11e8-89c0-97661a98658e.png)
![screenshot from 2018-11-16 12-54-11](https://user-images.githubusercontent.com/3481311/48646961-4ca8c180-e99f-11e8-8b25-7f2a419c8410.png)

Users still kept asking us?

Reply:
Well, the OpenShift version is certainly putting it more front-and-center ;).  That should help.  #677 and #691 were both suggesting a "register for a Tectonic plan" step.  Is that still required for OpenShift?  I'd expect `try.openshift.com` to be enough user input to get that information, but I'm not sure what the from-scratch flow looks like (presumably there's an accept-terms intermediate?  Maybe more?).  @smarterclayton, are there screenshots for a from-scratch registration somewhere?

Reply:
> ... but I'm not sure what the from-scratch flow looks like (presumably there's an accept-terms intermediate? Maybe more?).

So I just went through this with a different email address and GitHub auth, and there are two intermediate pages.  The first is a form for personal information and term-acceptance:

![form](https://user-images.githubusercontent.com/209920/48754311-1476dc80-ec46-11e8-9b8b-9bf25d860ba8.png)

The next is an email-confirmation page:

![confirm](https://user-images.githubusercontent.com/209920/48754205-b2b67280-ec45-11e8-9f88-7fcf780bb074.png)

Clicking on the confirmation link in the email took me to the page I posted [above][1], with the token front and center.  So I don't think the new flow has anything like the old flow's "register for a Tectonic plan" step.  Is that enough to back up the text I have here now?  If not, what additional text would you like to see?

[1]: https://github.com/openshift/installer/pull/663#discussion_r234339298